frontend/src/utils.js

```diff
@@ -311,3 +322,36 @@ export const getExtentFromGeoJSON = features => {
 
   return feature[0]?.getGeometry()?.getExtent()
 }
+
+/**
+ * Merge objects with a key to array structure
+ * Example: {
+ *   {
+ *     '2018-05-11': [],
+ *     '2018-05-16': []
+ *   }
+ * @param {Object} objectOne - First object
+ * @param {Object} objectTwo - Second object
+ * @return {Object} The merged object
+ */
+export function mergeObjects (objectOne, objectTwo) {
```

Comment:
Seems like you don't like utility libraries like lodash !? :) curious about that

Reply:
Mhm j'avoue que j'aurais pu utiliser lodash... J'ai un peu de mal à rajouter des libs pour une feature mais bon c'est du code en moins de notre côté c'est vrai